setup.py

```diff
@@ -0,0 +1,27 @@
+import setuptools
+
+with open("README.md", "r") as fh:
+    LONG_DESCRIPTION = fh.read()
+
+setuptools.setup(
+    name="agpypeline",
+    version="0.0.1",
+    author="Jacob van der Leeuw",
+    author_email="jvanderleeuw@email.arizona.edu",
+    description="Installable package for entrypoint and environment code within a transformer",
+    long_description="Package containing entrypoint.py from entrypoint code and transformer_class.py "
+                     "from environment code. This installable package allows for transformers to have"
+                     "common entrypoint and environment code in an installable package",
+    long_description_content_type="text/markdown",
+    url="https://github.com/AgPipeline/agpypeline",
+    packages=setuptools.find_packages(),
+    classifiers=[
+        "Programming Language :: Python :: 3",
```

Comment:
We actually need Python 3.7 or later. Be sure to test out whatever changes you make to this file